preload/resources/preload_helper.js

```diff
@@ -16,6 +16,11 @@ function verifyNumberOfResourceTimingEntries(url, number)
     assert_equals(numEntries, number, url);
```

Comment:
Nit: You could use the helper function here

Reply:
Will do.